### ISSUE TITLE
Remove {imagedir} suffix (and use the built-in imagesdir attribute)

### ISSUE THREAD

**Issue by Mogztter** [neo4j-contrib/training-v3#17]
> The path to the location of images is controlled by the imagesdir document attribute. The value of this attribute (empty by default) is added to the beginning of every image target. Therefore, you never need to reference this attribute. You only need to set it.

https://asciidoctor.org/docs/user-manual/#setting-the-location-of-images

**Before**

```
image::{imagedir}/GraphComponents.png[GraphComponents,width=400,align=center]
```

**After**

```
image::GraphComponents.png[GraphComponents,width=400,align=center]
```

**Comment by ElaineRosenberg** [comment]
Sounds good to me. I will change the sources

**Comment by Mogztter** [comment]
fixed in 7108a503d08b961d56c7db71ea23e8d7a9095155
